src/Chapter1.hs

```diff
@@ -619,8 +633,14 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let
+        absNumber = abs n
+        lastNumber = mod absNumber 10
+        secondLastNumber = div (mod absNumber 100 - lastNumber) 10
```

Comment:
Great!
For `div` the last digit won't matter, so it is okay not to subtract it in here 🙂